controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTrigger.java

```diff
@@ -200,7 +200,7 @@ private boolean changesAvailable(Application application, JobStatus previous, Jo
 
         JobStatus.JobRun previousSuccess = previous.lastSuccess().get();
         JobStatus.JobRun nextSuccess = next.lastSuccess().get();
-        if (previousSuccess.revision().isPresent() &&  ! previousSuccess.revision().get().equals(nextSuccess.revision().get()))
+        if (previousSuccess.revision().isPresent() && ! previousSuccess.revision().equals(nextSuccess.revision()))
```

Comment:
Not sure if nextSuccess.revision() can be empty(), but doesn't hurt to be sure.

Reply:
It can't but you are right!